src/http/server/handlers/static_file_handler.cr

```diff
@@ -115,10 +115,7 @@ class HTTP::StaticFileHandler
   end
 
   private def redirect_to(context, url)
-    context.response.status = :found
-
-    url = URI.encode_path(url.to_s)
-    context.response.headers.add "Location", url
+    context.response.redirect url
```

Comment:
Apparently this is where the `Path` comes from. It is hard to say whether this justifies the use of unrestricted `Path`s in the public API.

Reply:
At least I now remember the reason for adding `Path` to the restriction 😅
But I think I'd still stick to the plan and just call `to_s` here.